packages/frontapp/src/app/pages/HomePage.tsx

```diff
@@ -78,7 +78,11 @@ const HomePage: FC = () => {
                 <TitleArticle>Don't forget to pass</TitleArticle>
                 <TextBase>List of items you have taken and due dates</TextBase>
               </Wrapper>
-              <ListBooks items={data?.getAllTakenItems} />
+              <ListBooks
+                items={data?.getAllTakenItems.filter(
+                  (item) => item?.status !== 'Free'
+                )}
+              />
```

Comment:
I could be wrong, but i thought that this gql query already returns not free books